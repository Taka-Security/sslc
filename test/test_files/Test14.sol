contract A {
  enum EnumMedium { 
    el1,
    el2,
    el3,
    el4,
    el5,
    el6,
    el7,
    el8,
    el9,
    el10,
    el11,
    el12,
    el13,
    el14,
    el15,
    el16,
    el17,
    el18,
    el19,
    el20,
    el21,
    el22,
    el23,
    el24,
    el25,
    el26,
    el27,
    el28,
    el29,
    el30,
    el31,
    el32,
    el33,
    el34,
    el35,
    el36,
    el37,
    el38,
    el39,
    el40,
    el41,
    el42,
    el43,
    el44,
    el45,
    el46,
    el47,
    el48,
    el49,
    el50,
    el51,
    el52,
    el53,
    el54,
    el55,
    el56,
    el57,
    el58,
    el59,
    el60,
    el61,
    el62,
    el63,
    el64,
    el65,
    el66,
    el67,
    el68,
    el69,
    el70,
    el71,
    el72,
    el73,
    el74,
    el75,
    el76,
    el77,
    el78,
    el79,
    el80,
    el81,
    el82,
    el83,
    el84,
    el85,
    el86,
    el87,
    el88,
    el89,
    el90,
    el91,
    el92,
    el93,
    el94,
    el95,
    el96,
    el97,
    el98,
    el99,
    el100,
    el101,
    el102,
    el103,
    el104,
    el105,
    el106,
    el107,
    el108,
    el109,
    el110,
    el111,
    el112,
    el113,
    el114,
    el115,
    el116,
    el117,
    el118,
    el119,
    el120,
    el121,
    el122,
    el123,
    el124,
    el125,
    el126,
    el127,
    el128,
    el129,
    el130,
    el131,
    el132,
    el133,
    el134,
    el135,
    el136,
    el137,
    el138,
    el139,
    el140,
    el141,
    el142,
    el143,
    el144,
    el145,
    el146,
    el147,
    el148,
    el149,
    el150,
    el151,
    el152,
    el153,
    el154,
    el155,
    el156,
    el157,
    el158,
    el159,
    el160,
    el161,
    el162,
    el163,
    el164,
    el165,
    el166,
    el167,
    el168,
    el169,
    el170,
    el171,
    el172,
    el173,
    el174,
    el175,
    el176,
    el177,
    el178,
    el179,
    el180,
    el181,
    el182,
    el183,
    el184,
    el185,
    el186,
    el187,
    el188,
    el189,
    el190,
    el191,
    el192,
    el193,
    el194,
    el195,
    el196,
    el197,
    el198,
    el199,
    el200,
    el201,
    el202,
    el203,
    el204,
    el205,
    el206,
    el207,
    el208,
    el209,
    el210,
    el211,
    el212,
    el213,
    el214,
    el215,
    el216,
    el217,
    el218,
    el219,
    el220,
    el221,
    el222,
    el223,
    el224,
    el225,
    el226,
    el227,
    el228,
    el229,
    el230,
    el231,
    el232,
    el233,
    el234,
    el235,
    el236,
    el237,
    el238,
    el239,
    el240,
    el241,
    el242,
    el243,
    el244,
    el245,
    el246,
    el247,
    el248,
    el249,
    el250,
    el251,
    el252,
    el253,
    el254,
    el255,
    el256,
    el257,
    el258,
    el259,
    el260
  }
  
  struct StructA {
    uint240 var1;
    uint256 var2;
    EnumMedium var3;
  }
}