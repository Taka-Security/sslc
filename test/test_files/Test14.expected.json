[
    {
        "file": "Test14.sol",
        "contract": "A",
        "struct": "StructA",
        "slotsSaved": 1,
        "timedout": false
    }
]